egs2/cmu_arctic/tts1/conf/tuning/train_transformer_pre_sid.yaml

```diff
@@ -0,0 +1,92 @@
+# This configuration is for ESPnet2 to train Transformer-TTS. Compared to the
+# original paper, this configuration additionally use the guided attention
+# loss to accelerate the learning of the diagonal attention. It requires
+# 4 GPUs with 32 GB memory and it will finish within 1 day to finish the
+# training on Tesla V100.
+
```

Comment:
If you add your original modification to the config, it is better to mention about it here.
Otherwise, we should use the same naming as the other recipes.

Reply:
Added a comment about pretrain modification.